client/dashboard/me/security-connected-apps/index.tsx

```diff
@@ -152,6 +153,12 @@ export default function SecurityConnectedApps() {
 				/>
 			}
 		>
+			<ComponentViewTracker
+				eventName="calypso_dashboard_security_connected_apps_page_view"
+				properties={ {
+					total_items: totalItems,
+				} }
+			/>
```

Comment:
This is an interesting example where we can't use the usual `calypso_page_view` event because we're adding an extra property to it. Do we know why we're recording this property though? If we want to know how many connected apps the average user has for example, we'd just do a database query to look that up 🤔